### PR TITLE
Fixes #1763 - Refactor complex condition and allow creation menu secondary items appear even if no primary items are passed

### LINKED ISSUE #1763: Management Toolbar Creation Menu: Doesn't display secondary items if primary items are not defined

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: packages/clay-management-toolbar/src/ClayManagementToolbar.soy [hunk 1]
```diff
@@ -462,7 +462,15 @@
...
-	{if $creationMenu and (($creationMenu.primaryItems and length($creationMenu.primaryItems) == 1 and (not $creationMenu.secondaryItems or length($creationMenu.secondaryItems) == 0)) or ($creationMenu.secondaryItems and length($creationMenu.secondaryItems) == 1 and (not $creationMenu.primaryItems or length($creationMenu.primaryItems) == 0)))}
+	{let $isPrimaryItems: $creationMenu and $creationMenu.primaryItems and length($creationMenu.primaryItems) > 0 /}
+
+	{let $isSecondaryItems: $creationMenu and $creationMenu.secondaryItems and length($creationMenu.secondaryItems) > 0 /}
+
+	{let $isOnePrimaryItem: $isPrimaryItems and length($creationMenu.primaryItems) == 1 /}
+	
+	{let $isOneSecondaryItem: $isPrimaryItems and length($creationMenu.primaryItems) == 1 /}
+
+	{if ($isOnePrimaryItem and not $isSecondaryItems) or ($isOneSecondaryItem and not $isPrimaryItems)}
```
Comment (wincent) [inline packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: This is a huge improvement!

From a grammatical perspective, these names might work better ("is-plural" sounds like it violates concordance, but "has-plural" does not):

- `$isPrimaryItems` -> `$hasPrimaryItems`
- `$isSecondaryItems` -> `$hasSecondaryItems`
- `$isOnePrimaryItem` -> `$hasOnePrimaryItem` 
- `$isOneSecondaryItem` -> `$hasOneSecondaryItem`

Comment (carloslancha) [reply packages/clay-management-toolbar/src/ClayManagementToolbar.soy]: Changed, thx @wincent !

**Comment by rolandpakai** [comment]
I think this is a copy-paste issue:

`{let $hasOneSecondaryItem: $hasPrimaryItems and length($creationMenu.primaryItems) == 1 /}`

it should be:

`{let $hasOneSecondaryItem: $hasSecondaryItems and length($creationMenu.secondaryItems) == 1 /}`

**Comment by carloslancha** [comment]
Resent https://github.com/liferay/clay/pull/1766

**Pull request closed without merging**